python/cudf/cudf/benchmarks/bench_cudf_io.py

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2018-2020, NVIDIA CORPORATION.
+
+import pytest
+import cudf
+import glob
+
+@pytest.mark.parametrize("file_path", glob.glob("cudf/benchmarks//cuio_data/datasets/avro_*"))
```

Comment:
Maybe a higher level question, do we want to benchmark from files and capture the disk IO time or from memory buffers?

Reply:
Think memory buffers would be better, @vuule how are we doing it in libcudf ?

Reply:
Currently the cuIO benchmarks use buffers, but I am actually reconsidering this as of this morning.
The primary purpose of these benchmarks is to catch regressions. In that sense, using files is better.
However, if we want to know the GPU throughput, we need to use buffer input.

Maybe we should defalut to file input (for future CI purposes) and have a flag to switch to buffer input?

Reply:
But I am not sure if we can read compressed data file, as I am getting errors with read.
```
'utf-8' codec can't decode byte 0x8b in position 1: invalid start byte
```

Reply:
Update: I wasn't reading them properly, now both of them work. By default it is in file read mode, if you want to switch to memory buffers, you can use `--use_file False` while running pytest.